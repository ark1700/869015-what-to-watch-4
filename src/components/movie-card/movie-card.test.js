import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card.jsx";

const movieCard = {
  name: `Bohemian Rhapsody`,
  previewImage: `img/bohemian-rhapsody.jpg`,
};

it(`Render Main`, () => {
  const tree = renderer
    .create(<MovieCard
      movieCard={movieCard}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
