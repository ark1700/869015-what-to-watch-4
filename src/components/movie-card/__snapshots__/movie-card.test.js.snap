// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Movie card 1`] = `
<article
  className="small-movie-card catalog__movies-card"
  onMouseEnter={[Function]}
  onMouseLeave={[Function]}
>
  <div
    className="small-movie-card__image"
  >
    <video
      height="175"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="./movie-page"
    >
      The Grand Budapest Hotel
    </a>
  </h3>
</article>
`;
